### PR TITLE
Ldapadmin return json 1115

### PR DESCRIPTION
**Pull request by vdorut**
https://github.com/georchestra/georchestra/issues/1115

### CODE REVIEW

**Comment by vdorut** [comment]
runtime tested

**Comment by landryb** [comment]
Do we even need to set the Content-Type header in the curl GET example ? Since by default json is returned.. i think Content-Type is needed only for POST. Other that that, reads good to me, and thanks for working on it :)

**Comment by fvanderbiest** [comment]
LGTM, please merge.

File: ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/users/UserListResponse.java [hunk 1]
```diff
@@ -23,6 +25,7 @@
  * @author Mauricio Pazos
  *
  */
+@RequestMapping(produces = "application/json")
```
Comment (pmauduit) [inline ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/users/UserListResponse.java]: charset ?

File: ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/users/UsersController.java [hunk 2]
```diff
@@ -140,7 +140,7 @@ public void findAll( HttpServletRequest request, HttpServletResponse response )
 	 * @param response Returns the detailed information of the user as json
 	 * @throws IOException
 	 */
-	@RequestMapping(value=REQUEST_MAPPING+"/*", method=RequestMethod.GET)
+	@RequestMapping(value=REQUEST_MAPPING+"/*", method=RequestMethod.GET, produces = "application/json")
```
Comment (pmauduit) [inline ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/users/UsersController.java]: charset ?

**Comment by pmauduit** [comment]
Does not work after more thoroughly checks, needs rework

**Pull request merged**